toko/apps/products/views.py

```diff
@@ -1,18 +1,54 @@
-from django.shortcuts import render
+from operator import __or__ as OR
+
+from django.db.models import Q
 from django.views.generic.detail import DetailView
 from django.views.generic.list import ListView
-from .models import Product
+
+from .models import Product, Category
+
+
+class CategoryDetailView(DetailView):
+    """Return category detail."""
+
+    template_name = 'categories/detail.html'
+    model = Category
+    context_object_name = 'object_category'
```

Comment:
ini di karnakan jika mengukan jika objectnya category pemagilaan lopingnya
jadi kayaak gini {% for cat in  category.categories.all %}
jadi ambigue

On Thu, Feb 18, 2016 at 8:53 PM, Sutrisno Efendi <notifications@github.com>
wrote:

> In toko/apps/products/views.py
> <https://github.com/ular/toko/pull/7#discussion_r53315619>:
>
> >  from django.views.generic.detail import DetailView
> >  from django.views.generic.list import ListView
> > -from .models import Product
> > +
> > +from .models import Product, Category
> > +
> > +
> > +class CategoryDetailView(DetailView):
> > +    """Return category detail."""
> > +
> > +    template_name = 'categories/detail.html'
> > +    model = Category
> > +    context_object_name = 'object_category'
>
> object name-nya cukup category aja
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ular/toko/pull/7/files#r53315619>.
>